setup.py

```diff
@@ -58,11 +58,12 @@ def get_version(rel_path):
         "regex",
         "rich",
         "dm-tree",
+        "tf-nightly",
```

Comment:
We will still want `tensorflow-text` for our stable releases. Is there a way we can update `pip_build.py` to allow switching between the two?

Something like this above...
```
if is_nightly:
    ...
     setup_contents = setup_contents.replace(
         '"tensorflow-text"', '"tensorflow-text-nightly", "tf-nightly"'
     )
    ...
```

That way our repo is ready to build stable and nightly releases at anytime, and we don't have to add extra steps to our release cut branching process.

Reply:
Done